grp/glzmodmz.gi

```diff
@@ -170,7 +170,10 @@ local t,geni,m,slmats,gens,f,rels,i,j,k,l,mat,mat1,mats,id,nh;
     od;
   od;
 
-  return Group(slmats);
+  g := Group(slmats);
+  mat := Concatenation(id{[nh+1..n]},-id{[1..nh]});
+  SetInvariantBilinearForm(g,rec(matrix:=mat));
+  return g;
 end);
```

Comment:
Yes -- indeed, writing the tests for `Sp(2*n, Integers mod p^k)` lead me to discover that the invariant form was missing for `p=2`, and hence I added this to make the tests pass. so this was TDD (test driven development) :-).